<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-LUGISBJ" failures="1" tests="16" name="org.example.testRunners.R01_Smoke" time="164.441" errors="1" timestamp="2022-07-12T17:10:04 EET" skipped="0">
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="8.663"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="8.368"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="8.321"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="8.552">
    <failure message="The following asserts failed:
	URL login expected [https://demo.nopcommerce.com/] but found [https://demo.nopcommerce.com/login?returnurl=%2F]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
URL login expected [https://demo.nopcommerce.com/] but found [https://demo.nopcommerce.com/login?returnurl=%2F]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at org.example.stepDefinitions.SD02_Login.step4(SD02_Login.java:32)
at ✽.user could login successfully and go to home page(file:///C:/Users/qqError404/IdeaProjects/E-CommerceAutomationProject/src/main/resources/features/F09_WishList.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="9.449">
    <error message="Index 1 out of bounds for length 1" type="java.lang.IndexOutOfBoundsException">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 1 out of bounds for length 1
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at org.example.stepDefinitions.SD08_FollowUs.opens_new_tab(SD08_FollowUs.java:21)
at ✽."https://demo.nopcommerce.com/new-online-store-is-open" is opened in new tab(file:///C:/Users/qqError404/IdeaProjects/E-CommerceAutomationProject/src/main/resources/features/F08_FollowUs.feature:20)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="11.435"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="8.842"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="7.971"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="9.446"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="9.635"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="8.046"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="9.045"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="8.217"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="30.129"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="8.974"/>
  <system-out/>
  <testcase classname="org.example.testRunners.R01_Smoke" name="runScenario" time="9.348"/>
  <system-out/>
</testsuite> <!-- org.example.testRunners.R01_Smoke -->
